src/api/tokenList/tokenList.json

```diff
@@ -1,5 +1,6 @@
 [
   {
+    "id": 1,
```

Comment:
While it's ok when we add tokens ourselves in a fixed order, id won't necessarily match across networks

Reply:
The static token list will (potentially) always have the same index, since the deployment script uses this JSON to add tokens to the contract, after adding OWL as the first token.

So, as long as the contract is deployed using this list and the list is not re-ordered in any way, the ids remain the same for every network.